ios/Classes/SwiftWebviewCookieManagerPlugin.swift

```diff
@@ -96,33 +97,37 @@ public class SwiftWebviewCookieManagerPlugin: NSObject, FlutterPlugin {
         })
     }
     
-    public static func getCookies(url: String?, result: @escaping FlutterResult) {
-        let cookieList: NSMutableArray = NSMutableArray()
-        httpCookieStore!.getAllCookies { (cookies) in
-            for cookie in cookies {
-                if url == nil {
-                    cookieList.add(_cookieToDictionary(cookie: cookie))
-                }
-                else if cookie.domain.contains(URL(string: url!)!.host!) {
-                    cookieList.add(_cookieToDictionary(cookie: cookie))
-                }
+    public static func getCookies(urlString: String?, result: @escaping FlutterResult) {
```

Comment:
The change to this function was to fix a fatal crash I was getting when passing in empty string or invalid url string:

```
Fatal error: Unexpectedly found nil while unwrapping an Optional value: file /Users/navgattu/development/flutter/.pub-cache/git/webview_cookie_manager-431cc148be85167980ea71c3a8f066af3af02759/ios/Classes/SwiftWebviewCookieManagerPlugin.swift, line 106
```